awswrangler/timestream.py

```diff
@@ -9,11 +9,17 @@
 import boto3
 import pandas as pd
 from botocore.config import Config
+from botocore.loaders import Loader
+from botocore.model import ServiceModel
 
 from awswrangler import _data_types, _utils
 
 _logger: logging.Logger = logging.getLogger(__name__)
 
+_BOTOCORE_LOADER = Loader()
+_TIMESTREAM_JSON_MODEL = _BOTOCORE_LOADER.load_service_model(service_name="timestream-write", type_name="service-2")
```

Comment:
Just curious is it possible to reuse the service model loaded by boto itself and avoid loading it again? If not, I would prefer not to do this as it adds an overhead purely for validating the arguments which will be anyhow done by boto at later stage

Reply:
Unsure I can get to what boto is loading for itself, however I hear your point that boto will handle arguments validation anyway. I'm ok removing it entirely. This brings another question though, should we remove it from [s3/_fs.py](https://github.com/aws/aws-sdk-pandas/blob/main/awswrangler/s3/_fs.py#L38) too eventually?

Reply:
Good point, I wasn't aware we were doing that in the first place... IMO we should remove it as well but I understand this will be a breaking change for the users that pass some unused args that are filtered out and ignored